src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ReferralRepository.kt

```diff
@@ -10,17 +10,20 @@ import java.util.UUID
 
 interface ReferralRepository : JpaRepository<Referral, UUID> {
   @Query(
-    value = """select referralId, sentAt,
+    value = """select 
+      referralId, 
+      sentAt,
 			referenceNumber,
-			assignedToUserName,
 			interventionTitle,
+      dynamicFrameworkContractId,
```

Comment:
We don't use this anywhere in `ServiceProviderSentReferralSummaryDTO`. Does this have to be here?

Reply:
yes, we use in ReferralAccessFilter 
(I'm just writing tests for these)